backend/bin/deal-observer-backend.js

```diff
@@ -110,6 +111,9 @@ const sparkApiSubmitDealsLoop = async (pgPool, { sparkApiBaseUrl, sparkApiToken,
         point.intField('interval_ms', SPARK_API_SUBMIT_DEALS_LOOP_INTERVAL)
         point.intField('duration_ms', dt)
       })
+      recordTelemetry('submitted_deals_stats', point => {
+        point.intField('number_of_submitted_deals', numberOfSubmittedDeals)
```

Comment:
I believe this is precise enough:

```suggestion
        point.intField('submitted_deals', numberOfSubmittedDeals)
```

or

```suggestion
        point.intField('deals_submitted', numberOfSubmittedDeals)
```

Reply:
I'm going to go with the first suggestion as second one sounds like a boolean value.